test/unit/processing/siti-agri/build-invoice-lines.test.js

```diff
@@ -89,4 +89,28 @@ describe('Build invoice lines', () => {
     expect(console.error).toHaveBeenLastCalledWith('Invoice line is invalid. "deliveryBody" with value "RP" fails to match the required pattern: /^[A-Z]{2}\\d{2}$/')
     expect(invoiceLineIsValid.result).toBe(false)
   })
+
+  test('should exclude net lines', async () => {
+    invoiceLines.push({
+      schemeCode: 'SITIELM',
+      accountCode: 'ABC123',
+      fundCode: 'ABC12',
+      description: 'N00 - Net value of claim',
+      value: 100,
+      convergence: true,
+      deliveryBody: 'RP00'
+    })
+    const invoiceLinesParse = buildInvoiceLines(invoiceLines)
+    expect(invoiceLinesParse).toMatchObject([
+      {
+        schemeCode: 'SITIELM',
+        accountCode: 'ABC123',
+        fundCode: 'ABC12',
+        description: 'G00 - Gross value of claim',
```

Comment:
Why do we expect description to be 'G00 - Gross value of claim'? Does it swap the 'N00' description with the 'G00' description?

Reply:
In the test setup step at the top of the file it adds a G00 line.  In this test, I'm adding a N00 line to that existing array.

So the expectation is that we only get the G00 line and the N00 is removed.

Reply:
Oh okay - thanks for explaining :)